### ISSUE TITLE
Creating a Form in the Forms screen.

### ISSUE THREAD

**Issue by paaljoachim** [munirkamal/gutenberg-forms#97]
I made a test form in the Gutenberg Forms -> Forms screen.

<img width="1011" alt="Screen Shot 2020-04-13 at 21 35 03" src="https://user-images.githubusercontent.com/5323259/79153839-ce885e80-7dce-11ea-92f5-5d94e2c04077.png">

I to a page where I plan on using the Form.

<img width="516" alt="Screen Shot 2020-04-13 at 21 37 04" src="https://user-images.githubusercontent.com/5323259/79153909-ef50b400-7dce-11ea-9fc9-01d3ff00bed2.png">

Click **Insert Form**

<img width="912" alt="Screen Shot 2020-04-13 at 21 37 58" src="https://user-images.githubusercontent.com/5323259/79153968-10190980-7dcf-11ea-8d67-343d8f590204.png">

Saved Forms should be the first option that is open like so.
IF there is a Saved Form. If there are no saved forms then it opens showing the first category area as it does right now.

<img width="912" alt="Screen Shot 2020-04-13 at 21 38 41" src="https://user-images.githubusercontent.com/5323259/79154029-28892400-7dcf-11ea-967e-5cab05ab912b.png">

When a form is saved it would be nice to have Saved Forms open first to show the existing forms the user has saved. If user does not plan to use Saved forms then he/she can move on to one of the categories.

The result of clicking **Insert** shows not the form but a shortcode.
<img width="1009" alt="Screen Shot 2020-04-13 at 21 41 42" src="https://user-images.githubusercontent.com/5323259/79154312-9d5c5e00-7dcf-11ea-8c01-728b998592c2.png">

Bottom line a few things to fix.
1. If there is a saved form have Saved Forms option show up first.
2. Do not show shortcode but show the form. If one needs to refresh. Then add a refresh icon to the toolbar the user can click to have the form show up. As the user might want to adjust the form.

**Comment by munirkamal** [comment]
If you are creating a form in the forms screen (CPT) than you don't have access to saved forms. Because this is the place where you create your fresh forms and save them to be reusable anywhere.

The saved forms are only accessible when you are creating forms directly from a post/page/any other CPT and not from the Gutenberg Forms CPT.

The first suggestion will be implemented.

**Comment by ZafarKamal123** [comment]
Now added server side rendering for the saved forms preview instead of shortcode block ! 👍.
